src/include/storage/tile_group_header.h

```diff
@@ -228,6 +234,24 @@ class TileGroupHeader : public Printable {
                                         transaction_id);
   }
 
+  /*
+  The following method use Compare and Swap to set the tilegroup's
+  immutable flag to be true. 
+  */
+  inline bool SetImmutability() {
```

Comment:
In reality, the `inline` keyword is superfluous here. But, since everything else is marked inline, better to be consistent.